weaknesses/MASVS-CODE/MASWE-0076.md

```diff
@@ -1,38 +1,79 @@
+
+!!! quote "Google's [Using SDKs safely and securely](https://support.google.com/googleplay/android-developer/answer/13326895)"
+
+    "App developers are required to treat any data collection from within their app by an SDK as if they collected it directly."
+
+!!! quote "Apple's [Third-party SDK requirements](https://developer.apple.com/support/third-party-SDK-requirements/)"
+
+    "When you use a third-party SDK with your app, you are responsible for all the code the SDK includes in your app, and need to be aware of its data collection and use practices.
+
+For more information on privacy and data collection declarations, see @MASWE-0112.
 
 ## Impact
 
 Using dependencies with known vulnerabilities in mobile apps can result in various security risks, including but not limited to:
 
-- **Data Exposure**: Attackers can exploit known vulnerabilities in dependencies to gain unauthorized access to sensitive user data or app functionality. This can lead to data breaches that expose private information, such as user credentials or personal data. Such breaches can have legal and financial consequences for the app owner and undermine user trust.
-- **Compromise of System Integrity and Functionality**: Vulnerabilities in dependencies may allow attackers to compromise the app's overall integrity, potentially introducing malicious behavior such as unauthorized code execution or privilege escalation. This can lead to account takeover or app downtime.
-- **Non-Compliance**: Using outdated or vulnerable dependencies may result in noncompliance with security standards and regulations. This can expose businesses to regulatory penalties and legal liabilities, especially those in industries that handle sensitive data, such as healthcare or finance.
+- **Sensitive Data Exposure**: Vulnerable dependencies may be exploited to bypass access controls or cryptographic protections, which could lead to the exposure of sensitive user data, including credentials, session tokens, and personally identifiable information (PII). This can result in data breaches, which can have legal, financial and reputational consequences.
+- **Execution of Unauthorized Code or Privilege Escalation**: Exploitable flaws in embedded dependencies can allow attackers to execute arbitrary code within the app's context (e.g., through code injection or remote code execution (RCE)), escalate privileges, or manipulate app behavior. The overall impact can range from full compromise of user accounts, abuse of backend services or persistent access to protected resources. The consequences for the business can be severe, including financial loss, service disruption, and damage to customer trust.
```

Comment:
```suggestion
- **Execution of Unauthorized Code or Privilege Escalation**: Exploitable vulnerabilities in embedded dependencies can allow attackers to execute arbitrary code within the app's context (e.g., through code injection), escalate privileges, or manipulate app behavior. The overall impact can range from full compromise of user accounts, abuse of backend services or persistent access to protected resources. The business impact can be severe, including financial loss, service disruption, and damage to customer trust.
```
The 'remote' part of RCE is a bit ambiguous since the injection could come from the app itself / another app. Feels weird to have RCE for that.